packages/react-components/src/components/styles/TogetherMode.styles.ts

```diff
@@ -198,16 +198,26 @@ export const togetherModeParticipantDisplayName = (
   isParticipantHovered: boolean,
   participantSeatingWidth: number,
   color: string
-): CSSProperties => {
-  const MIN_DISPLAY_NAME_WIDTH = 100;
+): React.CSSProperties => {
+  const width =
+    isParticipantHovered || participantSeatingWidth * REM_TO_PX_MULTIPLIER > 100
+      ? 'fit-content'
+      : _pxToRem(0.7 * participantSeatingWidth * REM_TO_PX_MULTIPLIER);
```

Comment:
Given the complexity of the CSS in the file can we name the 0.7 magic number here? want to understand better what the multiplication here is calculating